ci/cloudbuild/builds/check-api.sh

```diff
@@ -51,11 +51,37 @@ cmake --install cmake-out
 function dump_abi() {
   local library="$1"
   local prefix="$2"
+  local public_headers="${prefix}/include/google/cloud/${library#google_cloud_cpp_}"
+  if [[ "${library}" == "google_cloud_cpp_common" ]] || [[ "${library}" == "google_cloud_cpp_grpc_utils" ]]; then
```

Comment:
It is simpler to use expression disjunction rather than command disjunction.  That is, ...
    ``[[ exp1 || exp2 ]]``
instead of ...
    ``[[ exp1 ]] || [[ exp2 ]]``

Reply:
Done.